sw/host/opentitanlib/src/io/gpio.rs

```diff
@@ -22,6 +22,8 @@ pub struct PinConfiguration {
     /// The default/initial analog level of the pin in Volts, has effect only in `AnalogOutput`
     /// mode.
     pub volts: Option<f32>,
+    /// Whether the pin is inverted when we write to it. This feature can be used to account transistor on the pin line.
+    pub invert: bool,
```

Comment:
Please could you add a comment here stating that it's currently only used in the chipwhisperer?